### ISSUE TITLE
your menu button is does not meet accessibility standards.

### ISSUE THREAD

**Issue by timcombs** [henrikwirth/gatsby-starter-wordpress-twenty-twenty#7]
looks like a problem with a show-modal class on the menu that needs to be removed with the menu is closed.

**Comment by henrikwirth** [comment]
Hi @timcombs thanks for the info.

Since I'm using this project for some performance tests of a thesis right now, this wasn't my focus yet. I'll try to update the theme as soon as I have some more time.

Happy to welcome PRs though if they reflect the behaviour of the php-version of the theme.

✌️
